src/egon/data/datasets/gas_grid.py

```diff
@@ -660,6 +661,33 @@ def insert_gas_pipeline_list(
     )
 
 
+def remove_isolated_gas_buses():
+    """Delete gas buses which are not connected to the gas grid.
+
+    Returns
+    -------
+    None.
+    """
+    targets = config.datasets()["gas_grid"]["targets"]
+
+    db.execute_sql(
+        f"""
+        DELETE FROM {targets['buses']['schema']}.{targets['buses']['table']}
+        WHERE "carrier" = 'CH4'
+        AND scn_name = 'eGon2035'

```

Comment:
Is the problem about isolated gas buses happening only in the scenario eGon2035 or does it make sense for you to use this function also for the eGon100RE scenario?

Reply:
The buses gas buses of eGon100RE are a copy a the gas buses of eGon2035 so we do not need to apply this function for the eGon100RE scenario.

Reply:
Thanks for the clarification.